### PR TITLE
Thumbnail demo control

### PR DESCRIPTION
**Pull request by KetpuntoG**
fixing thumbnail

### CODE REVIEW

File: demonstrations/tutorial_optimal_control.metadata.json [hunk 1]
```diff
@@ -20,11 +20,7 @@
         },
         {
             "type": "large_thumbnail",
-            "uri": "/_static/large_demo_thumbnails/thumbnail_large_tutorial_optimal_control.png"
-        },
-        {
-            "type": "hero_image",
-            "uri": "/_static/hero_illustrations/optimal_control_hero.png"
+            "uri": "/_static/large_demo_thumbnails/thumbnail_large_optimal_control.png"
```
Comment (isaacdevlugt) [inline demonstrations/tutorial_optimal_control.metadata.json:23]: The path to the image is `demonstrations/optimal_control/thumbnail_large_optimal_control.png`. Is this right?

**Review by isaacdevlugt — APPROVED** [review]

File: demonstrations/tutorial_optimal_control.metadata.json (hunk 1 above)
Comment (KetpuntoG) [reply demonstrations/tutorial_optimal_control.metadata.json:23]: Oh wait, thanks! I should put the image in other place

**Review by ikurecic — APPROVED** [review]
lgtm, compared to the JSON of a recent demo

File: demonstrations/tutorial_optimal_control.metadata.json (hunk 1 above)
Comment (KetpuntoG) [reply demonstrations/tutorial_optimal_control.metadata.json:23]: the image was already in place, the only file I had to change was the metadata 👍

**Pull request merged**